build-system/tasks/compile.js

```diff
@@ -421,6 +423,7 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
     }
 
     let stream = gulp.src(srcs)
+        .pipe(gulpIf(isThirdPartyFile, gulpStringReplace('/**', '/* ')))
```

Comment:
We already have a gulp replace, see https://github.com/ampproject/amphtml/pull/21830/files#diff-b2216d09c55c9ee1f2f2c879bee3b619R439

Also, can you verify that this is working correctly? I tried to do replacement before running through closure in https://github.com/ampproject/amphtml/pull/21818, and it didn't work.

Reply:
@jridgewell See description.